cclient/response/get_neighbors.c

```diff
@@ -11,14 +11,19 @@ void neighbor_info_t_copy(void* _dst, const void* _src) {
   neighbor_info_t *dst = (neighbor_info_t*)_dst, *src = (neighbor_info_t*)_src;
   dst->address = src->address;
   dst->all_trans_num = src->all_trans_num;
-  dst->invalid_trans_num = src->invalid_trans_num;
+  dst->random_trans_req_num = src->random_trans_req_num;
   dst->new_trans_num = src->new_trans_num;
+  dst->invalid_trans_num = src->invalid_trans_num;
+  dst->stale_trans_num = src->stale_trans_num;
+  dst->sent_trans_num = src->sent_trans_num;
+  dst->connectionType = src->connectionType;
```

Comment:
sank_cast

Reply:
sorry I don't get it.
what is sank_cast?

Reply:
Typo, Sam meant snake_case.
`connection_type` instead of `connectionType `

Reply:
oh got it!